packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts

```diff
@@ -924,9 +924,10 @@ type MessagesType = {
   'modals.interest.withdrawal.balance1': 'Your {coinTicker} Balance'
   'modals.interest.withdrawal.enteramount': 'Enter withdrawal amount'
   'modals.interest.withdrawal.loading': 'Doing work...'
-  'modals.interest.withdrawal.recap': 'You are requesting to withdraw {withdrawalAmountFiat} ({withdrawalAmountCrypto}) from your Interest Account. After confirming this withdrawal, you will not continue to earn interest on the amount withdrawn.'
+  'modals.interest.withdrawal.recap': 'You are requesting to withdraw <b>{withdrawalAmountFiat}</b> ({withdrawalAmountCrypto}) from your Interest Account. After confirming this withdrawal, you will not continue to earn interest on the amount withdrawn.'
```

Comment:
Is it a good idea to add HTML here? https://formatjs.io/docs/getting-started/installation/#minimal-application

Maybe you could use the `values` to add this `<b>`

Reply:
This is common thing we use around the app, since we use translations on places where we need it to be like that I think we are fine. In case that we want to use some more generic way we might introduce this logic with values in such case for example case where we want to use same label with `<b>` and `<i>` but to be honest that is over engineering imho.